src/arvtool.c

```diff
@@ -394,6 +399,164 @@ arv_tool_control (int argc, char **argv, ArvDevice *device)
         }
 }
 
+static void
+arv_tool_network (int argc, char **argv, ArvDevice *device)
+{
+        if (!ARV_IS_GV_DEVICE (device))
+                printf ("This is not a GV device\n");
+        else if (argv[2] != NULL) {
```

Comment:
It would be nice if by default `network` alone displays the network configuration

Reply:
For now, the `ip` shows the persistent IP configuration, which is different from the **current** configuration. Maybe we should show both?

Reply:
Yes